public/app/core/components/Footer/Footer.tsx

```diff
@@ -17,42 +18,68 @@ export interface FooterLink {
 
 export let getFooterLinks = (): FooterLink[] => {
   // @PERCONA
-  return [
-    {
-      id: 'pmm-dump',
-      text: 'PMM Dump',
-      icon: 'brain',
-      url: '/graph/pmm-dump',
-    },
-    {
-      id: 'pmm-logs',
-      text: 'PMM Logs',
-      icon: 'download-alt',
-      url: '/logs.zip',
-      target: '_blank',
-    },
-    {
-      target: '_blank',
-      id: 'pmm-docs',
-      text: t('nav.help/documentation', 'Documentation'),
-      icon: 'document-info',
-      url: 'https://per.co.na/pmm_documentation',
-    },
-    {
-      target: '_blank',
-      id: 'support',
-      text: t('nav.help/support', 'Support'),
-      icon: 'question-circle',
-      url: 'https://per.co.na/pmm_support',
-    },
-    {
-      target: '_blank',
-      id: 'community',
-      text: t('nav.help/community', 'Community'),
-      icon: 'comments-alt',
-      url: 'https://per.co.na/pmm_community',
-    },
-  ];
+  if (isPmmAdmin(config.bootData.user) && config.bootData.user.isGrafanaAdmin) {
```

Comment:
The check for `config.bootData.user.isGrafanaAdmin` is redundant here as it's included in the `isPmmAdmin`.